app/environments/brassbirmingham/brassbirmingham/envs/classes/buildings/building.py

```diff
@@ -0,0 +1,75 @@
+class Building:
+    """
+    Building object
+
+    :param name: any of [goods, cotton, pottery]
+    :param tier: ex: 3
+    :param cost: cost
+    :param coalCost: ex: 1
+    :param ironCost: ironCost
+    :param victoryPointsGained: victory points gained from selling
+    :param incomeGained: income gained from selling
+    :param networkPoints: amount of points each road gets during counting step
+    :param canBeDeveloped=True:
+    :param onlyPhaseOne=False:
+    :param onlyPhaseTwo=False:
+    """
+
+    def __init__(
+        self,
+        type,
+        name,
+        tier,
+        cost,
+        coalCost,
+        ironCost,
+        victoryPointsGained,
+        incomeGained,
+        networkPoints,
+        canBeDeveloped,
+        onlyPhaseOne,
+        onlyPhaseTwo,
```

Comment:
nit: Since a building can't be both `onlyPhaseOne` and `onlyPhaseTwo`, we could consolidate these into a single variable like `restrictedBuildPhase` which either takes on `None`, `1`, or `2`